tech/languages/fortran/fortran-installation.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Fortran
+subsection: fortran
+section: tech-languages
+version: 11.1.1
+order: 1
+description: A general-purpose, imperative computer programming language, especially suited to numeric computation and scientific computing.
+---
+# Fortran installation and usage
+
+## gfortran installation
+Fortran is a compiled language and as such it needs a compiler. To install gfortran
+compiler, simply type:
+
+```
+$ sudo dnf install gcc-gfortran
+```
+
+This command should download and install package with compiler called gfortran. To compile
+and link a program written in Fortran, simply run the `gfortran` command like this:
+
+```
+gfortran your_source.f
+```
+
+This will produce a binary file created from your source code. By default, this binary
+file is named `a.out`.
+You can run this binary by typing:
+
+```
+$ ./a.out
```

Comment:
I think this should be in sync with the sample `your_source.f` file.

Reply:
Hello, I have added the compiler option to specify the output file name.